swagger.yaml

```diff
@@ -207,7 +207,7 @@ paths:
                 type: object
                 properties:
                   server_time:
-                    type: number
+                    type: integer
                     example: 1603400958947
```

Comment:
Still fails here, which is clearly a bug since the reader is deserializing this as `String` ... but it's clearly a valid `Int64`. 
I'll ignore the failing test on my side until they fix the reader.